tapdb/universe_stats.go

```diff
@@ -53,6 +68,16 @@ type UniverseStatsStore interface {
 	// universe/
 	QueryUniverseAssetStats(ctx context.Context,
 		arg UniverseStatsQuery) ([]UniverseStatsResp, error)
+
+	// QueryAssetStatsPerDaySqlite returns the stats for a given asset
+	// grouped by day in a SQLite specific format.
+	QueryAssetStatsPerDaySqlite(ctx context.Context,
+		q AssetStatsPerDayQuery) ([]AssetStatsPerDay, error)
+
+	// QueryAssetStatsPerDayPostgres returns the stats for a given asset
+	// grouped by day in a Postgres specific format.
+	QueryAssetStatsPerDayPostgres(ctx context.Context,
+		q AssetStatsPerDayQueryPg) ([]AssetStatsPerDayPg, error)
```

Comment:
Why do we have backend specific query functions here? I thought that wasn't necessary with a unix timestamp.

Reply:
The available time calculation functions are different between the backends (see the two different queries).
We know that we'll run into such differences here and in `lnd` in the future anyway, so this kind of serves as a proof of concept for backend specific queries.